source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2476,6 +2476,49 @@ export class ImageSet extends CardElementContainer {
     }
 }
 
+export class CaptionSource extends SerializableObject {
```

Comment:
@paulcam206 and @dclaux 

CaptionSource and MediaSource are nearly the same except getSchemaKey and render. What's the strategy for this? Create a base class like mimeSource, and then make CaptionSource and MediaSource subclasses of it, or duplicate the code like in this file

Reply:
Yes, it would be good to introduce a base `ContentSource` class and and have both `MediaSource` and the new `CaptionSource` classes extend it, only overriding `getSchemaKey` and `render`.

Reply:
Makes sense! Done